skore/src/skore/_config.py

```diff
@@ -94,6 +112,13 @@ def config_context(
     show_progress : bool, default=None
         If True, show progress bars. Otherwise, do not show them.
 
+    plot_backend : {"matplotlib", "plotly"}, default="matplotlib"
```

Comment:
```suggestion
    plot_backend : {"matplotlib", "plotly"}, default=None
```

I don't think it is the right place to mention that the default backend is matplotlib.